src/sinks/blackhole/config.rs

```diff
@@ -19,13 +23,16 @@ pub struct BlackholeConfig {
     /// The number of seconds between reporting a summary of activity.
     ///
     /// Set to `0` to disable reporting.
-    #[derivative(Default(value = "1"))]
+    #[derivative(Default(value = "default_print_interval_secs()"))]
     #[serde(default = "default_print_interval_secs")]
-    pub print_interval_secs: u64,
+    #[serde_as(as = "serde_with::DurationSeconds<u64>")]
+    #[configurable(metadata(docs::examples = 10))]
+    pub print_interval_secs: Duration,
 
     /// The number of events, per second, that the sink is allowed to consume.
```

Comment:
Thoughts on a `unit_type` of "events per second"? or some short-handed version of units

Reply:
good call, added this.